composer.json

```diff
@@ -8,7 +8,7 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.1 || ^8.0",
```

Comment:
Hier geen `^7.2` i.p.v. `^7.1`?

Reply:
In de andere bundles hadden we het meer "nodig" om een PHPUnit te upgraden naar tenminste versie 8. Dat is hier niet zo, maar het zou alsnog kunnen...

Reply:
v2 van deze bundle is ook een specifieke Sf 4 versie. Dus er is geen project dat <7.4 draait en alle andere bundles (met een specifieke Sf 4 versie) hebben ook ^7.2. Dus op zich wel netjes...

Reply:
Ja, eens. Verder zijn die versies toch allemaal hartstikke EOL dus het zou verder ook niet uit maken.